Makefile

```diff
@@ -49,18 +50,19 @@ all-variants: linux-amd64 darwin-amd64 windows-amd64
 .PHONY: linux-amd64
```

Comment:
(GitHub won't let me comment on the right line) Can you adjust the `$(LOCAL_BINARY)` target as well?

Reply:
Added $VERSION and $GITCOMMIT_SHA to both targets. Fixed in d22b66e